qt/preferences_dialog.cpp

```diff
@@ -81,6 +81,16 @@ namespace qt
       });
     }
 
+    QCheckBox * transliterationCheckBox = new QCheckBox("Transliterate to Latin");
+    {
+      transliterationCheckBox->setChecked(framework.LoadTransliteration());
+      connect(transliterationCheckBox, &QCheckBox::stateChanged, [&framework](int i)
+      {
+        framework.SaveTransliteration(static_cast<bool>(i));
+        framework.AllowTransliteration(static_cast<bool>(i));
```

Comment:
nit: I prefer explicit ```bool const enable = i > 0;```

Reply:
In other places, it's still static cast. For a one-liner it's clearer IMO.